src/Farmer/Builders/Builders.Vm.fs

```diff
@@ -245,6 +254,10 @@ type VirtualMachineBuilder() =
     [<CustomOperation "custom_data">]
     member _.CustomData(state:VmConfig, customData:string) = { state with CustomData = Some customData }
 
+    [<CustomOperation "linux_configuration">]
```

Comment:
I think other builders usually have `add_linux_configurations` for lists and `add_linux_configuration` for single objects.

Reply:
👍

Reply:
done